wdl/gwas/regenie.wdl

```diff
@@ -46,7 +19,7 @@ workflow regenie {
         }
     }
 
-    call coding_gather {
-        input: files=sub_step2.coding
+    call summary.regenie_summary as post_summary {
```

Comment:
uh oh this is gonna be shitton of localization.... but the import wdl I meant that we have the 'tasks' i.e. functions there so we don't have the summary task in multiple places, when adding the summarization workflow to pregiven sumstats.  I would do the post-processing of single sumstats within the step2.  import `summary` task from a separate wdl file (I assume that this is possible without having any workflows there???)

Reply:
Hmm, is there any excess localization here which wouldn't happen before? Anyways the single sumstats would have to be first localized to the summary task and to put them all together would require localizing them all in the single task, right? Or am I missing something?

Also, I was under the assumption that it is only possible to import and call workflows only, not just the tasks. I'll need to have a look if that works.

Maybe a topic for discussion in the sw meeting.

Reply:
Aaa yeah it seems to be possible to call imported tasks directly without first having to call the workflow. Or at least womtool validate seemed to be ok with it. This simplifies things...

Reply:
> 
yea previously the summarization happened in a single phenotype step. Now you are summarizing all of the full summarystats which is gonna be localizing terabyte+ to a single machine.

Reply:
we want the summary to happen in a single pheno step so then we can just gather the summaries, which is gonna be localizing very small files per pheno

Reply:
Actually since it's calling the subworkflow which has as first step the summary task in a scatter, it doesn't localize them all on a single machine but rather follows the scatter.